### PR TITLE
Basic layout 

### PR DESCRIPTION
**Pull request by dreyfus92**
Simple layout fixes to start using TailwindCSS integration.

### CODE REVIEW

**Comment by trvswgnr** [comment]
this looks good! can we resolve the merge conflicts? (sorry that was me)

currently the orange on dark gray for dark mode is a little "halloween-y" to me. how do you feel about changing the background to a deep grayish blue? pretty much what github's dark mode is. something like #121627.

for the dark mode toggle, i think that if we detect a preference for either dark mode or light mode then we should just use that, and avoid the clutter. if there's no preference detected then we can default to light mode and show the toggle. what do you think?

**Comment by dreyfus92** [comment]
Good idea, I will apply those suggestions as soon as I can 😁

File: src/components/BaseHead.astro [hunk 1]
```diff
@@ -1,17 +1,15 @@
 ---
-// Import the global.css file here so that it is included on
-// all pages through the use of the <BaseHead /> component.
-import '../styles/global.css';
+import "../styles/global.css";
```
Comment (trvswgnr) [inline src/components/BaseHead.astro:2]: @dreyfus92 did you add rules somewhere that enforces double-quotes?

File: src/components/BaseHead.astro [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
 
 export interface Props {
-	title: string;
-	description: string;
-	image?: string;
+  title: string;
+  description: string;
+  image?: string;
```
Comment (trvswgnr) [inline src/components/BaseHead.astro:7]: is this change to 2 spaces from your editor settings or does astro have a style guide?

**Review by trvswgnr — CHANGES_REQUESTED** [review]
the changes look good!

i'm not sure we need to change the code style and formatting. if astro has a guide in place then let's use that, otherwise let's agree on some rules and enforce them everywhere.

thanks for doing this!!

File: src/components/BaseHead.astro (hunk 1 above)
Comment (dreyfus92) [reply src/components/BaseHead.astro:2]: Astro's Lang support extension for VSCode comes with Formatting powered by prettier so I tend to use astro's code formatter. Here's the link of the extension https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode

File: src/components/BaseHead.astro (hunk 2 above)
Comment (dreyfus92) [reply src/components/BaseHead.astro:7]: It's pretty much the formatter.

**Review by trvswgnr — APPROVED** [review]

**Pull request merged**